.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,7 +1,7 @@
 contact_links:
   - name: Report a Security Vulnerability
     url: https://github.com/space-wizards/space-station-14/blob/master/SECURITY.md
-    about: Please report security vulnerabilities privately so we can fix them before they are publicly disclosed.
-  - name: Request a Feature
-    url: https://discord.gg/rGvu9hKffJ
-    about: Submit feature requests on our Discord server (https://discord.gg/rGvu9hKffJ).
+    about: Please report security vulnerabilities to the Space Wizards privately so they can fix them before they are publicly disclosed.
+  - name: Toolshed Feature Request
```

Comment:
Interesting link. Any reason we're not linking, say, engine changes as well? 
Figure that's too obscure?

(I had to put GitHub into split mode to read this, my brain simply *was not* parsing the diff)